starter-code/index.html

```diff
@@ -0,0 +1,253 @@
+                                    culpa,
+                                    dicta impedit? Quidem, ipsa nisi.</p>
+
+                            </div>
+
+                        </div>
+                        <div class="col-md-3">
+                            <div class="margin20">
+                                <img src="./img/slider-3.jpg" alt="">
+                                <h3>asdasdasd asdasdasd oooomomd fdsfdsfsdf sdfsdfsdfsd fsdfsdfsd </h3>
+                                <p>Posted on 5 April</p>
+                                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Ad possimus aliquam totam
+                                    quidem
+                                    sapiente exercitationem quisquam nobis consequuntur ratione, molestiae dolorem,
+                                    magni
+                                    dolore sequi nesciunt maxime natus excepturi ex rerum!</p>
+                            </div>
+                        </div>
+                        <div class="col-md-3">
+                            <div class="margin20">
+                                <img src="./img/slider-4.jpg" alt="">
+                                <h3>asdaskldjaskld kljkljasdas ñlajsdasdas inknaskln</h3>
+                                <p>Posted on 8 May</p>
+                                <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Natus rerum odit quas
+                                    itaque
+                                    eum maxime ipsum nisi laboriosam accusamus corrupti. Dolorem enim, libero temporibus
+                                    omnis reiciendis maxime aut animi neque.</p>
+                            </div>
+                        </div>
+                    </div>
+                </article>
+            </div>
+        </section>
+
+
+    </main>
+
+    <footer>
+
+        <section class="footersct1 padding40">
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-3">
+                        <article>
+                            <h3>Revera</h3>
+                            <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellendus ut repudiandae rem
+                                eius ex
+                                provident excepturi. Harum, modi cupiditate sed assumenda temporibus veniam quas enim,
+                                voluptatibus
+                                ducimus sint sit aperiam!</p>
+                        </article>
+                    </div>
+                    <div class="col-md-3">
+                        <article>
+                            <h3>Meta</h3>
+                            <a href="">Log In</a>
+                            <a href="">Entries</a>
+                            <a href="">Comments</a>
+                            <a href="">Wordpress</a>
```

Comment:
siempre que pongáis un enlace que no vaya a ningún lado tenéis que poner href="#"